### PR TITLE
Updated Docs Theme

### PR DESCRIPTION
**Pull request by alexwolfe**
Hey Guys, 

This PR updates the theme for the docs to match the current website. It also has a few bug fixes in it, improvement to the typography, and ux improvements. Thanks.

### CODE REVIEW

**Comment by benarent** [comment]
image

Docs looks great.

I would be good to provide a nested nav, in the sidebar and breadcrumbs, but we can come back to that in the future

File: docs/4.3.yaml [hunk 1]
```diff
@@ -48,32 +48,41 @@ nav:
...
         #    - OneLogin:  enterprise/sso/ssh_one_login.md
         #    - OIDC:  enterprise/sso/oidc.md
         #    - Okta:  enterprise/sso/ssh_okta.md
+        - Approval Workflow: enterprise/workflow/index.md
+        #- Teleport Plugins:
+        #    - Teleport  Plugin Setup: enterprise/workflow/ssh_approval_jira_cloud.md
+        #    - Teleport Jira Server Plugin Setup: enterprise/workflow/ssh_approval_jira_server.md
+        #    - Teleport Mattermost Plugin Setup: enterprise/workflow/ssh_approval_mattermost.md
+        #    - Teleport Pagerduty Plugin Setup: enterprise/workflow/ssh_approval_pagerduty.md
+        #    - Teleport Slack Plugin Setup: enterprise/workflow/ssh_approval_slack.md
```
Comment (benarent) [inline docs/4.3.yaml:73]: Commented these out, we can add nestested nav back in the future.

**Review by benarent — APPROVED** [review]

**Comment by webvictim** [comment]
`milv` is complaining: https://drone.gravitational.io/gravitational/teleport/320/2/2

**Review by webvictim — CHANGES_REQUESTED** [review]
Please fix `milv` complaints before we merge: https://drone.gravitational.io/gravitational/teleport/320/2/2

**Pull request merged**